docs/reference/feature-matrix.md

```
@@ -28,7 +28,7 @@ This document provides a comprehensive compatibility matrix for key Dynamo featu
 | **LoRA** | | | ✅ | [K8s Guide][lora] |
 | **Tool Calling** | ✅ | ✅ | ✅ | [Tool Calling Doc][tools] |
 | **Speculative Decoding** | 🚧 | ✅ | ✅ | Backend READMEs |
-| **Dynamo Snapshot** | ✅ | | ✅ | [Snapshot Docs][snapshot] |
+| **Dynamo Snapshot** | ✅ | 🚧 | ✅ | [Snapshot Docs][snapshot] |
```

markdownlint-cli2 (0.22.1)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: "snapshot"

(MD052, reference-links-images)